scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -327,10 +350,22 @@ export class AuthoringIntegrationWrapper extends React.PureComponent<IPropsWrapp
                 <Nav.SideBarTabs
                     activeTab={this.state.sideWidget?.id}
                     onActiveTabChange={(val) => {
+                        if (val == null && closedThroughAction.closed == false) {
+                            closedThroughAction.closed = true;
+                        }
+
+                        const isWidgetPinned = (() => {
+                            if (this.state.sideWidget?.id == val && this.state.sideWidget.id != null) {
```

Comment:
`this.state.sideWidget?.id == val` is double equals instead triple intentional?

Reply:
Not intentional

Reply:
can you use triple equals then?